src/components/advancedResources/AdvancedResourcesCard.tsx

```diff
@@ -0,0 +1,12 @@
+import ResourcesCard from "../ResourcesCard";
```

Comment:
the parent folder shouldn't be name `advancedResources` it should instead be named sections

Reply:
fixed